docs/old-reference-guide/modules/ROOT/pages/conversion.adoc

```diff
@@ -0,0 +1,431 @@
+
+[NOTE]
+====
+**Axon 5 Change**: The `Serializer` concept has been replaced with `Converter`.
+
+Converters are responsible for changing the technical representation of message payloads. While they are primarily used for messages, converters are not directly tied to the messaging infrastructure and can be used for any data conversion needs.
+====
+
+By default, **no conversion takes place** in Axon Framework. Message payloads remain in their original Java object form.
+
+When using **Spring Boot**, if Jackson is on the classpath, Axon will automatically configure a `JacksonConverter` as the default converter. This converter uses https://github.com/FasterXML/jackson[Jackson] to convert objects to and from JSON format, which produces a compact representation suitable for network transmission and storage.
+
+You may also implement your own converter by creating a class that implements `Converter` and configuring it within Axon's configuration.
+
+== Converter Implementations
+
+Axon Framework provides several `Converter` implementations for different use cases:
+
+=== JacksonConverter
+
+The `JacksonConverter` uses https://github.com/FasterXML/jackson[Jackson] to convert objects to and from JSON format.
+This is the most commonly used converter and is automatically configured when using Spring Boot with Jackson on the classpath.
+
+JSON is compact, human-readable, and widely supported, making it ideal for:
+- Event storage (when using event sourcing)
+- Command and query messages (in distributed systems)
+- Interoperability between different systems
+
+[source,java]
+----
+import org.axonframework.conversion.json.JacksonConverter;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class ConverterConfiguration {
+
+    public Converter buildConverter() {
+        return new JacksonConverter(); // Uses default ObjectMapper
+    }
+
+    public Converter buildCustomConverter() {
+        ObjectMapper customMapper = new ObjectMapper();
+        // Configure your ObjectMapper as needed
+        customMapper.findAndRegisterModules();
+        return new JacksonConverter(customMapper);
+    }
+}
+----
+
+=== `AvroConverter`
+
+The `AvroConverter` uses https://avro.apache.org/[Apache Avro] to convert objects to and from https://avro.apache.org/docs/1.12.0/specification/#binary-encoding[binary single-object-encoded format].
+
+Avro provides:
+- Compact binary format
+- Schema-based conversion with strong typing
+- Schema evolution support
+- Ideal for event storage and messaging where schema enforcement is desired
+
+The `AvroConverter` requires a `SchemaStore` to operate, which must be able to resolve schemas for all messages being processed.
+
+[source,java]
+----
+import org.axonframework.conversion.avro.AvroConverter;
+import org.axonframework.conversion.json.JacksonConverter;
+
+public class ConverterConfiguration {
+
+    public Converter buildAvroConverter(SchemaStore schemaStore) {
```

Comment:
imports of the example are missing the import for what I guess should be `org.apache.avro.message.SchemaStore`, would add that for clarity